flink/src/test/java/org/apache/iceberg/flink/TestFlinkSchemaUtil.java

```diff
@@ -252,7 +252,7 @@ public void testInconsistentTypes() {
         Types.BinaryType.get(), new VarBinaryType(VarBinaryType.MAX_LENGTH),
         new VarBinaryType(100), Types.BinaryType.get());
     checkInconsistentType(
-        Types.TimeType.get(), new TimeType(6),
+        Types.TimeType.get(), new TimeType(),
```

Comment:
Are these related changes, or should the time changes go into a separate PR?

Reply:
I will create one.

Reply:
Created #1521

Reply:
Merged #1521.